app/models/concerns/sufia/generic_file/metadata.rb

```diff
@@ -0,0 +1,99 @@
+module Sufia
+  module GenericFile
+    module Metadata
+      extend ActiveSupport::Concern
+
+      included do
+        property :label, predicate: ActiveFedora::RDF::Fcrepo::Model.downloadFilename, multiple: false
+
+        property :depositor, predicate: ::RDF::URI.new("http://id.loc.gov/vocabulary/relators/dpt"), multiple: false do |index|
+          index.as :symbol, :stored_searchable
+        end
+
+        property :arkivo_checksum, predicate: ::RDF::URI.new('http://scholarsphere.psu.edu/ns#arkivoChecksum'), multiple: false
+
+        property :relative_path, predicate: ::RDF::URI.new('http://scholarsphere.psu.edu/ns#relativePath'), multiple: false
+
+        property :import_url, predicate: ::RDF::URI.new('http://scholarsphere.psu.edu/ns#importUrl'), multiple: false do |index|
+          index.as :symbol
+        end
+
+        property :part_of, predicate: ::RDF::DC.isPartOf
+        property :resource_type, predicate: ::RDF::DC.type, multiple: false do |index|
```

Comment:
Only changed this one line to add multiple:false, this is not an optimal solution, breaks upgrade compatibility.